pkg/sliexporter/vshnpostgresql_controller/vshnpostgresql_controller.go

```diff
@@ -102,7 +104,7 @@ func (r *VSHNPostgreSQLReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		}
 
 		// Create a pobe that will always fail
-		probe, err = probes.NewFailingPostgreSQL(vshnpostgresqlsServiceKey, inst.Name, inst.Namespace)
+		probe, err = probes.NewFailingPostgreSQL(vshnpostgresqlsServiceKey, inst.Name, inst.ObjectMeta.Labels["crossplane.io/claim-namespace"])
```

Comment:
You could put the label key in a constant. Maybe it's actually already in a common/util package somewhere...